meta/main.yml

```diff
@@ -11,6 +11,13 @@ galaxy_info:
         - 16.04
         - 18.04
   galaxy_tags:
-    - Strongswan
-    - IPSEC
-dependencies: []
+    - strongswan
+    - ipsec
+dependencies:
+  - role: onaio.monit
+    vars:
+      monit_scripts: "{{ ipsec_monit_scripts }}"
```

Comment:
@evansmurithi what's the default value for `ipsec_monit_scripts`? I think that should be the scripts we know be installed.

Reply:
Seen the vars.

Reply:
The default is `["monit", "ipsec"]`